src/solvers/petscdmlibmeshimpl.C

```diff
@@ -958,7 +958,7 @@ PetscErrorCode  DMCreate_libMesh(DM dm)
 
   PetscFunctionBegin;
   PetscValidHeaderSpecific(dm,DM_CLASSID,1);
-  ierr = PetscNewLog(dm,&dlm);CHKERRQ(ierr);
+  ierr = PetscNew(&dlm);CHKERRQ(ierr);
```

Comment:
No if-defing required here, eh? I guess `PetscNew` should have just been used whenever this was originally written...

Reply:
It was written in 2012, and Jed was recommending PetscNewLog to users [as late as 2016](https://scicomp.stackexchange.com/questions/23471/petsc-difference-between-petscnew-and-petscmalloc#comment37122_23471).  I assume they made `PetscNew` do more sophisticated logging without user assistance at some point since, but this code was fine at the time.